artview/parser.py

```diff
@@ -7,10 +7,10 @@
 import argparse
 
 # Get the version
-f = open("version", "r")
-NAME = f.readline().strip()
-VERSION = f.readline().strip()
-f.close()
+import version
```

Comment:
I don't see a version.py in the distribution. Was this supposed to incorporate the old script or is this a separate package?